docs/src/guides/advanced/dynamic-values.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Dynamic Values
+navTitle: Dynamic Values
+description: Defining and Resolving run-time aggregations
+order: 55
+group: 4-advanced
+---
+
+** DRAFT / WIP / DRAFT / WIP / DRAFT / WIP / DRAFT / WIP / DRAFT / WIP / DRAFT / WIP / DRAFT **
```

Comment:
It's ok to acknowledge the status of a guide or functions. However, as a dev I wouldn't know what to expect when this would not be 'draft' - and is that referring to the Guide or the functionality itself?

I also wonder if we need this much caveat as this guide is actually implemented in code. 

Perhaps there can be a standard 'Status' section that provides few statements on what is draft (the guide, the functions,...), what the feature can be used for now, and next steps to "v1"